docs/zigabee.md

```diff
@@ -0,0 +1,26 @@
+# Comparison to ZigBee
+This network layer is influenced by the design of ZigBee, but does not implement it
+directly.
+
+## Which is better?
+ZigBee is a much more robust, feature-rich set of protocols, with many different vendors
+providing compatible chips.
+
+RF24Network is cheap.  While ZigBee radios are well over $20, nRF24L01 modules can be found
+for under $2.  My personal favorite is [MDFly RF-IS2401](http://www.mdfly.com/index.php?main_page=product_info&products_id=82).
```

Comment:
hyperlink leads to a product that doesn't exist anymore ☹️ Alternatives?

Reply:
Maybe its best not to link to a specific product, unless somebody has a really good/preferred source.

Reply:
ok I removed the product link, but left the generic comparison of price.